<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabsMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabEvent">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Événement</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupAuth">
          <property name="title">
           <string>Authentification Eventbrite</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lblConnectedUser">
             <property name="text">
              <string>Non-connecté</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineApiKey">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::EchoMode::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Clé d'API privée</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnConnectUser">
               <property name="text">
                <string>Connecter utilisateur</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnSaveSession">
               <property name="text">
                <string>Sauvegarder session</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnLoadSession">
               <property name="text">
                <string>Charger session</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupEvent">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Organisation et événement Eventbrite</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="comboOrg">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Liste des organisations</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboEvent">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Liste des événements</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="btnLoadEvent">
               <property name="text">
                <string>Charger événement</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkOverwrite">
               <property name="text">
                <string>Écraser profils existants</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Informations sur l'événement</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="1">
              <widget class="QLabel" name="lblEventName">
               <property name="text">
                <string>N/A</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Nom</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="lblEventDate">
               <property name="text">
                <string>N/A</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Date</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>ID évén.</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="lblEventID">
               <property name="text">
                <string>N/A</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAttendees">
       <attribute name="title">
        <string>Participants</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableAttendees">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID Eventbrite</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Prénom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Rôle</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Compagnie</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Imprimé?</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFilters">
       <attribute name="title">
        <string>Filtres</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Remplacement manuel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QComboBox" name="comboManualFilter">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
               <property name="placeholderText">
                <string>Choisir propriété</string>
               </property>
               <item>
                <property name="text">
                 <string>Prénom</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Nom</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Rôle</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Compagnie</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnApplyManualReplacement">
               <property name="text">
                <string>Appliquer remplacements</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableManualFilter">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>0</number>
             </property>
             <property name="columnCount">
              <number>3</number>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Valeur actuelle</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nb occurences</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Valeur de remplacement</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Filtres automatiques</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>En construction...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabNametags">
       <attribute name="title">
        <string>Cocardes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnSelectDocTemplate">
            <property name="text">
             <string>Sélectionner gabarit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDocTemplate">
            <property name="text">
             <string>Gabarit choisi</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupAutoTemplate">
          <property name="title">
           <string>Remplissage des variables du gabarit</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QTableWidget" name="tableDocVariables">
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>200</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Variable</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Valeur de remplacement</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Étape de génération additionnelle : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboCustomGenerators">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Aucune</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="btnGenerateNametags">
               <property name="text">
                <string>Générer cocardes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkMarkPrinted">
               <property name="text">
                <string>Marquer comme imprimé</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
